index.bs

```diff
@@ -1257,8 +1257,55 @@ To <dfn>get or create a sandbox realm</dfn> given |name| and |navigable|:
 <div algorithm>
 To <dfn>create a sandbox realm</dfn> with |window|:
 
-Issue: Define creation of sandbox realm. This is going to return a
-{{SandboxWindowProxy}} wrapping |window|.
+1. Let |window environment settings| be the [=environment settings object=] whose
+   [=relevant global object=] is |window|.
+
+1. Let |global| be a new {{SandboxWindowProxy}} wrapping |window|.
+
+   Issue: This is the key step but it's basically totally undefined.
+
+1. Let |agent| be |window environment settings|'s [=realm=]'s <a spec=html for=realm>agent</a>.
+
+1. Let |execution context| be the result of [=creating a new realm=] given |agent| using
+   |global| as the host-defined global object.
+
+1. Let |realm| be the value of |execution context|'s Realm component.
+
+1. Let |settings object| be a new [=environment settings object=] whose algorithms are defined as follows:
+
+  <dl>
+    <dt>The [=realm execution context=].
+    <dd>Return |execution context|.
+
+    <dt>The [=environment settings object/module map=]
+    <dd>Return |global|'s module map.
+        Issue: Need to define module map for {{SandboxWindowProxy}} objects.
```

Comment:
Looks like this issue is not rendered properly? Perhaps it needs newlines or needs to be outside of dl.